website/cue/reference/components/transforms/base/log_to_metric.cue

```diff
@@ -85,8 +85,12 @@ base: components: transforms: log_to_metric: configuration: {
 				type: string: syntax: "template"
 			}
 			tags: {
-				description: "Tags to apply to the metric."
-				required:    false
+				description: """
+					Tags to apply to the metric.
+
+					Both keys and values are templateable, which enables you to attach dynamic tags to events.
```

Comment:
```suggestion
					Both keys and values can be templated, allowing you to attach dynamic tags to events.
```

Reply:
Hi, I update the docs in source code and re-generate the docs. Thank you for your advice!